README.md

```diff
@@ -118,9 +118,23 @@ You can access snapshot builds from the sonatype snapshot repository by adding t
 ```
 
 ## Build
+
+### Requirements
+
+* [Node.Js](https://nodejs.org/en/download/)
+* Java JDK >= 1.8
+* [Lombok Java](https://projectlombok.org) Library (debian pacakge `liblombok-java`)
```

Comment:
Lombok is defined in the maven dependencies. The extra debian package doesn't need to be installed for building... You only would need a lombok plugin for your favorite IDE.

Reply:
I think only if you're using Eclipse. Even then Eclipse seemed to struggle finding the right libraries.